frontend/www/js/omegaup/components/user/Profilev2.vue

```diff
@@ -0,0 +1,268 @@
+<template>
+  <div class="container-fluid p-0 mt-0">
+    <div class="row">
+      <div class="col-md-2">
+        <div class="card">
+          <img
+            class="m-1"
+            height="20"
+            v-bind:src="`/media/flags/${profile.country_id.toLowerCase()}.png`"
+            v-bind:title="profile.country_id"
+            v-if="profile.country_id"
+            width="25"
+          />
+          <div class="card-body">
+            <div class="img-thumbnail rounded-circle bottom-margin">
+              <img class="rounded-circle" v-bind:src="profile.gravatar_92" />
+            </div>
+          </div>
+          <div class="card-title text-center">
+            <div class="mb-3">
+              <omegaup-user-username
+                v-bind:classname="profile.classname"
+                v-bind:username="profile.username"
+              ></omegaup-user-username>
+            </div>
+            <div class="mb-3">
+              <h4 class="m-0">
+                {{ profile.rankinfo.rank &gt; 0 ? `#${profile.rankinfo.rank}`:
```

Comment:
```suggestion
                {{ profile.rankinfo.rank > 0 ? `#${profile.rankinfo.rank}`:
```

ya podemos usar `>` aquí!